### PR TITLE
task 5 GET /api/reviews/:review_id complete

### PR DESCRIPTION
**Pull request by souste**

### CODE REVIEW

**Review by yalicee — CHANGES_REQUESTED** [review]
Great work so far but what if we have a invalid id? is there any error handling functionality for that?

**Comment by souste** [comment]
Thanks Alex, I've amended the test

File: __tests__/next one [hunk 1]
```diff
@@ -5,46 +5,8 @@ app.use(express.json());
...
-  );
-};
-
 models.js;
 
-exports.selectReviewById = () => {
-  return db.query(// test on sql sheet);
-};
-
 app.test.js;
```
Comment (emilyb93) [inline __tests__/next one]: What is this file?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -91,3 +91,46 @@ describe("/api/reviews", () => {
...
+    return request(app)
+      .get(`/api/previews/${reviewId}`)
+      .expect(404)
+      .then((res) => {
+        expect((res) => {
+          expect(res.body.msg).toBe("Invalid path");
+        });
+      });
+  });
+});
```
Comment (emilyb93) [inline __tests__/app.test.js:136]: What if the rest of the URL was fine, but the review id was not a number like `/api/reviews/sports`

**Review by emilyb93 — CHANGES_REQUESTED** [review]
an extra test please :)

File: __tests__/next one (hunk 1 above)
Comment (souste) [reply __tests__/next one]: Sorry that was a temporary comment to remind myself to test the table in another file, I thought  copied over it.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -91,3 +91,57 @@ describe("/api/reviews", () => {
...
+  test("GET - 500 Internal Server Error", () => {
+    return request(app)
+      .get(`/api/reviews/bananas`)
+      .expect(500)
+      .then((res) => {
+        expect((res) => {
+          expect(res.body.msg).toBe("Internal Server Error");
+        });
+      });
+  });
```
Comment (yalicee) [inline __tests__/app.test.js]: These last two tests should be one 400 invalid id test with an id that is not a number

**Review by yalicee — APPROVED** [review]
See my comment on your test. I will approve as the rest of your code is good and it should not be a problem

File: __tests__/app.test.js (hunk 3 above)
Comment (souste) [reply __tests__/app.test.js]: That's great, thanks Alex!

**Pull request merged**